src/main/java/com/crowdin/cli/commands/actions/BundleDownloadAction.java

```diff
@@ -94,21 +97,25 @@ private BundleExport buildBundle(Outputter out, ClientBundle client, Long bundle
                 this.noProgress,
                 false,
                 () -> {
-                    BundleExport status = client.startExportingBundle(bundleId);
+                    BundleExport status = null;
+                    try {
+                        status = client.startExportingBundle(bundleId);
 
-                    while (!status.getStatus().equalsIgnoreCase("finished")) {
-                        ConsoleSpinner.update(String.format(RESOURCE_BUNDLE.getString("message.spinner.building_bundle_percents"), status.getProgress()));
-                        Thread.sleep(1000);
+                        while (!status.getStatus().equalsIgnoreCase("finished")) {
+                            ConsoleSpinner.update(String.format(RESOURCE_BUNDLE.getString("message.spinner.building_bundle_percents"), status.getProgress()));
+                            Thread.sleep(1000);
 
-                        status = client.checkExportingBundle(bundleId, status.getIdentifier());
+                            status = client.checkExportingBundle(bundleId, status.getIdentifier());
 
-                        if (status.getStatus().equalsIgnoreCase("failed")) {
-                            throw new RuntimeException(RESOURCE_BUNDLE.getString("message.spinner.build_has_failed"));
+                            if (status.getStatus().equalsIgnoreCase("failed")) {
+                                throw new RuntimeException(RESOURCE_BUNDLE.getString("message.spinner.build_has_failed"));
+                            }
                         }
-                    }
-
-                    ConsoleSpinner.update(String.format(RESOURCE_BUNDLE.getString("message.spinner.building_bundle_percents"), 100));
 
+                        ConsoleSpinner.update(String.format(RESOURCE_BUNDLE.getString("message.spinner.building_bundle_percents"), 100));
+                    } catch (MaxNumberOfRetriesException e) {
```

Comment:
Any reason why this is different from the same catch in `DownloadAction.java`? We should probably update the warning message there as well since there could be other reasons why it failed.

Reply:
This was missed in DownloadActions when adding handlers of other errors. Updated for DownloadAction as well.